dnf/logging.py

```diff
@@ -177,7 +180,7 @@ def _setup_file_loggers(self, logfile_level, verbose_level, logdir, log_size, lo
         logger_warnings.addHandler(handler)
 
         lr_logfile = os.path.join(logdir, dnf.const.LOG_LIBREPO)
-        libdnf.repo.LibrepoLog.addHandler(lr_logfile, verbose_level <= DEBUG)
+        libdnf.repo.LibrepoLog.addHandler(lr_logfile, logfile_level <= ALL)
```

Comment:
What this does do though (IIUC) is it changes when debug level messages get logged in the librepo log. Previously it was when `debuglevel >= 3` and now its `loglevel >= 10`, so that's a change in behavior.

We probably can't solve (well, improve :slightly_smiling_face: ) this without changing anything and I'm fine with changing "something". Probably should at least be documented in changelog / release notes?

Reply:
Yes it is a change in behavior. There is also a similar change in sack.py to hawkey log. My thinking was that if we have an option setting log file levels it should also change librepo and hawkey logs. I am all for explaining this in the changelog. 

I could also explicitly mention this in the man page section for `logfilelevel`, but changes like this feel weird to me since we will likely change it again in DNF5 in addition the current behavior is also quite new and not document anywhere either I think.

Reply:
I agree. The thing is this is a stopgap solution with the messy logging structure we have in place here.

I feel the same way WRT dnf-5, but there we should definitely log into a single log file (or come up with a different structure if needed, but this one makes little sense), ideally log to journald I think. So I think it's a good idea to mention it in the manpage under `logfilelevel` as well.

Reply:
Updated the manpage. :slightly_smiling_face: